### PR TITLE
[MU4] Fix MSVC compiler warnings

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
reg. 
* unreachable code (C4702)
* 'initializing': conversion from 'size_t' to 'int', possible loss of data (C4267)
* declaration hides class member (C4458)

### CODE REVIEW

File: src/engraving/libmscore/stafftype.cpp [hunk 1]
```diff
@@ -280,8 +280,6 @@ bool StaffType::isSimpleTabStaff() const
     default:
         return false;
     }
-
-    return false;
 }
```
Comment (cbjeukendrup) [inline src/engraving/libmscore/stafftype.cpp:283]: Removing this makes other compilers emit warnings. In my experience the universal fix is:
```c++
    default: 
        break;
    }

    return false;
}
```

Comment (Jojo-Schmitz) [reply src/engraving/libmscore/stafftype.cpp]: Done

Comment (Jojo-Schmitz) [reply src/engraving/libmscore/stafftype.cpp]: Doesn't even compile though, needs a `break;` in the `default:` case

Comment (cbjeukendrup) [reply src/engraving/libmscore/stafftype.cpp]: I did write `break;` 😉

Comment (Jojo-Schmitz) [reply src/engraving/libmscore/stafftype.cpp]: Outch, reading helps...

**Review by cbjeukendrup — APPROVED** [review]

**Pull request merged**